.idea/vcs.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="IssueNavigationConfiguration">
+    <option name="links">
+      <list>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="#(\d+)" />
+          <option name="linkRegexp" value="https://github.com/esamarathon/oengusio/issues/$1" />
```

Comment:
How does this function when it's a pull request?

Reply:
GitHub redirects depending on the type. Explained in 33a26757bfe541d818bc479ba63eeb836dc4dec9:

```
Note that GitHub automatically redirects, for example:

  https://github.com/esamarathon/oengusio/issues/68

to

  https://github.com/esamarathon/oengusio/pull/68

if the number refers to a pull request.
```

Reply:
Cool